core/src/test/java/org/infinispan/test/TestingUtil.java

```diff
@@ -887,6 +902,23 @@ public static boolean replaceInterceptor(Cache cache, CommandInterceptor replaci
    }
 
    /**
+    * Replaces an existing interceptor of the given type in the interceptor chain with a new interceptor
+    * instance passed
+    * as parameter.
+    *
+    * @param replacingInterceptor        the interceptor to add to the interceptor chain
+    * @param toBeReplacedInterceptorType the type of interceptor that should be swapped with the new one
+    * @return true if the interceptor was replaced
+    */
+   public static boolean replaceInterceptor(Cache cache, SequentialInterceptor replacingInterceptor, Class<? extends SequentialInterceptor> toBeReplacedInterceptorType) {
+      ComponentRegistry cr = extractComponentRegistry(cache);
+      // make sure all interceptors here are wired.
+      cr.wireDependencies(replacingInterceptor);
```

Comment:
No, we can't be sure, but wiring those interceptors is the responsibility of whomever added them, not this method's.